guides/common/modules/proc_importing-a-subscription-manifest-into-foreman-server.adoc

```diff
@@ -25,8 +25,10 @@ endif::[]
 endif::[]
 ifeval::["{mode}" == "disconnected"]
 * Ensure you have a Red{nbsp}Hat subscription manifest exported from the Red Hat Customer Portal.
+You will use the same manifest in
+xref:configuring-server-to-synchronize-content-over-a-network_{context}[].
 ifndef::orcharhino[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html/getting_started_with_rhel_system_registration/adv-reg-rhel-using-rhsm_#using_manifests_con[Using manifests for a disconnected Satellite Server] in _Subscription Central_.
+For more information, see {RHDocsBaseURL}subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_disconnected_satellite_server/index[Creating and managing manifests for a disconnected {ProjectServer}] in _Subscription Central_.
```

Comment:
```suggestion
For more information, see {RHDocsBaseURL}subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_disconnected_satellite_server/index[Creating and managing manifests for a disconnected Satellite Server] in _Subscription Central_.
```
For Foreman documentation, we would normally use the attribute. But in this case, the title of the referenced docs shouldn't change with the build, so let's hard code it.
The same below.